src/traces/parcoords/parcoords.js

```diff
@@ -557,7 +557,7 @@ module.exports = function(root, svg, styledData, layout, callbacks) {
                     .outerTickSize(2)
                     .ticks(wantedTickCount, d.tickFormat) // works for continuous scales only...
                     .tickValues(d.ordinal ? // and this works for ordinal scales
-                        sdom.map(function(d, i) {return texts && texts[i] || d;}) :
+                        sdom.map(function(d, i) {return texts && texts[i] !== undefined ? texts[i] : d;}) :
```

Comment:
I'd let `false` pass through (ie get stringified) - not super useful, but it would be weird to stringify `true` but turn `false` into some number. `null` and `undefined` don't have opposites in the same way.